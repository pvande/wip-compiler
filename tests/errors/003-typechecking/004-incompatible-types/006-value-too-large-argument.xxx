y := (a : byte) => {}
z := y(256)
