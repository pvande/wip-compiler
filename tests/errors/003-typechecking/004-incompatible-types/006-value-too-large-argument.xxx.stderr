

Error: "No overload for that function takes those argument types"
In [1;37mtests/errors/003-typechecking/004-incompatible-types/006-value-too-large-argument.xxx[0m on line [1;37m2[0m

> [0;36mz := y(256)[0m
  [0;31m       ^^^ [0m
